### ISSUE TITLE
Measure distance

### ISSUE THREAD

**Issue by shujaatak** [green-anger/GlobeViewer#54]
GlobeViewer is already a fantastic software and it would be awesome if distance measuring feature is added to the project.

**Comment by green-anger** [comment]
Agree, plus a lot of different things that can be useful. The thing is I'm not planning to put in much time on this project for the nearest future. I hope to return when I get a bit more knowledge on DOD (Data-Oriented Design) which can be a great help to process a lot of data. And map soft usually uses a lot of overlay information, such as markers, roots, etc. Meanwhile feel free to raise any issue with a functionality proposal. It'll be of great help to me in choosing development direction once I'm back.
